### ISSUE TITLE
Add label to the translation icon

### ISSUE THREAD

**Issue by camillaryd** [justarrived/just-match-frontend#700]
__Description__

Someone picked this icon to signify the automatic translation is on or off. This needs to be explained in words otherwise we don't understad or see it. See the sketch below.

![desktop](https://cloud.githubusercontent.com/assets/19821275/16801851/5c470a1a-48fd-11e6-8812-1bbd334c5c8f.png)

**Comment by buren** [comment]
I couldn't agree more, even I that now what it means have trouble seeing the difference in colour.

**Comment by buren** [comment]
@lenaar perhaps something that you'd like to add? :)

**Comment by pstenstrm** [comment]
We have the issue #576. Should we still do that in combination or close it for the benefit of this issue?

**Comment by lenaar** [comment]
I can do both of them I guess, they are bit different issues but for the same element, one is just add text alternative, another one is add color

**Comment by pstenstrm** [comment]
That's true, but I think both issues are ment to fix related problems. We wanted to change the color to pink because we couldn't really see if the icon was active or not.

I don't think we "need" to add the color when we add the text label, since the label should solve both issues.

**Comment by camillaryd** [comment]
I say start with the text label and make the link blue if the background is white to make it more visible.
